.github/workflows/upgrade.yaml

```diff
@@ -0,0 +1,66 @@
+name: upgrade
+on:
+  push:
+    paths:
+      #- "manifest_staging/charts/**"
```

Comment:
TODO is to uncomment this line and remove the following line before merging right?

Reply:
that's correct